README.md

```diff
@@ -4,3 +4,22 @@ AzureIntegration
 Features that integrate ASP.NET Core with Azure.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+
+SiteExtensions
+===
+
+To install a nightly preview of the ASP.NET Core runtime site extension for testing purposes:
+1. In the Azure portal select App Services -> your site -> Application settings
+1. Set `SCM_SITEEXTENSIONS_FEED_URL` application setting to `https://dotnet.myget.org/F/aspnetcore-release/`
+1. Go to `Advanced Tools` -> `Site extensions` -> `Gallery`
+1. Enter `AspNetCoreRuntime` into `Search` box and click `Search`
+1. Click `+` to install site extension, click `Restart site` on the right side of the page when installation finishes.
```

Comment:
Might help to mention (a) to wait until the animation completes (and user is prompted) before clicking `Restart site`, (b) `Restart site` only refreshes the installed extensions on the Kudu site, and (c) `DEVELOPMENT TOOLS` -> `Extensions` should now list the newly-installed extension.